packages/components/organisms/f-checkout/src/tenants/en-GB.js

```diff
@@ -114,9 +114,10 @@ const messages = {
             },
 
             DuplicateOrder: {
-                title: 'This looks like a duplicate order',
-                message: 'So you’re not charged twice, your order has not been processed',
-                buttonText: 'Ok'
+                title: 'This order has not been placed',
+                message: 'It’s the same as the one you made a moment ago and we want to check it’s not a mistake. Wait 15 mins or change an item in your order',
```

Comment:
Are we able to change the messaging on this to be a little more friendly?

`Please wait 15 mins or change an item in your order and try again soon.`

Reply:
No, straight from UX&D i'm afraid